entrypoints/options/utils.ts

```
@@ -0,0 +1,129 @@
+  }
+}
+
+function setNameFieldError({
+  target,
+  type,
+  nameCategory,
+  index,
+  errorType,
+}: SetFieldErrorInput) {
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const errorField: HTMLParagraphElement = document.querySelector(`#nameField-error-${nameCategory}-${String(index)}`)!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)